app/package.json

```diff
@@ -11,8 +11,14 @@
     "cypress:e2e": "cypress run --e2e"
   },
   "dependencies": {
+    "@bcgov/bc-sans": "^1.0.1",
```

Comment:
apparently this is being updated to v2 this week? watch out for it. @Bottle7 @BradyMitch

Reply:
Any way to find out if it will be breaking? I don't see any update info on their repo.

Reply:
doesn't appear to be breaking changes from the inside information that I'm reading, but I'm not sure if the communication is finalized or official yet. just keep an eye out.